Odata-docs/webapi-8/getting-started.md

```diff
@@ -148,16 +148,10 @@ namespace Lab01.Controllers
         }
 
         [EnableQuery]
-        public ActionResult<Customer> Get([FromRoute] int key)
```

Comment:
There's no reason for this change. `$expand` works okay even when the return type is `ActionResult<Customer>`.